[2025-03-31T13:18:59.082-0500] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-31T13:18:59.086-0500] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: exec_mult_tasks.taskA manual__2025-03-31T18:18:56.165989+00:00 [queued]>
[2025-03-31T13:18:59.089-0500] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: exec_mult_tasks.taskA manual__2025-03-31T18:18:56.165989+00:00 [queued]>
[2025-03-31T13:18:59.089-0500] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-03-31T13:18:59.232-0500] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): taskA> on 2025-03-31 18:18:56.165989+00:00
[2025-03-31T13:18:59.234-0500] {standard_task_runner.py:72} INFO - Started process 425227 to run task
[2025-03-31T13:18:59.235-0500] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'exec_mult_tasks', 'taskA', 'manual__2025-03-31T18:18:56.165989+00:00', '--job-id', '42', '--raw', '--subdir', 'DAGS_FOLDER/exec_multiple_tasks.py', '--cfg-path', '/tmp/tmp2db356ye']
[2025-03-31T13:18:59.236-0500] {standard_task_runner.py:105} INFO - Job 42: Subtask taskA
[2025-03-31T13:18:59.365-0500] {task_command.py:467} INFO - Running <TaskInstance: exec_mult_tasks.taskA manual__2025-03-31T18:18:56.165989+00:00 [running]> on host royal-07.cs.wisc.edu
[2025-03-31T13:18:59.622-0500] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='cloud.user' AIRFLOW_CTX_DAG_ID='exec_mult_tasks' AIRFLOW_CTX_TASK_ID='taskA' AIRFLOW_CTX_EXECUTION_DATE='2025-03-31T18:18:56.165989+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-31T18:18:56.165989+00:00'
[2025-03-31T13:18:59.622-0500] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-31T13:18:59.630-0500] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-03-31T13:18:59.630-0500] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', 'tmp1tcb1euv.sh']
[2025-03-31T13:18:59.633-0500] {subprocess.py:99} INFO - Output:
[2025-03-31T13:18:59.633-0500] {subprocess.py:106} INFO - TASK A has started!
[2025-03-31T13:18:59.633-0500] {subprocess.py:106} INFO - TASK A printing 1
[2025-03-31T13:18:59.633-0500] {subprocess.py:106} INFO - TASK A printing 2
[2025-03-31T13:18:59.633-0500] {subprocess.py:106} INFO - TASK A printing 3
[2025-03-31T13:18:59.634-0500] {subprocess.py:106} INFO - TASK A printing 4
[2025-03-31T13:18:59.634-0500] {subprocess.py:106} INFO - TASK A printing 5
[2025-03-31T13:18:59.634-0500] {subprocess.py:106} INFO - TASK A printing 6
[2025-03-31T13:18:59.634-0500] {subprocess.py:106} INFO - TASK A printing 7
[2025-03-31T13:18:59.634-0500] {subprocess.py:106} INFO - TASK A printing 8
[2025-03-31T13:18:59.634-0500] {subprocess.py:106} INFO - TASK A printing 9
[2025-03-31T13:18:59.634-0500] {subprocess.py:106} INFO - TASK A printing 10
[2025-03-31T13:18:59.634-0500] {subprocess.py:106} INFO - TASK A has ended!
[2025-03-31T13:18:59.634-0500] {subprocess.py:110} INFO - Command exited with return code 0
[2025-03-31T13:18:59.740-0500] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-31T13:18:59.741-0500] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=exec_mult_tasks, task_id=taskA, run_id=manual__2025-03-31T18:18:56.165989+00:00, execution_date=20250331T181856, start_date=20250331T181859, end_date=20250331T181859
[2025-03-31T13:19:00.009-0500] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-31T13:19:00.022-0500] {taskinstance.py:3901} INFO - 3 downstream tasks scheduled from follow-on schedule check
[2025-03-31T13:19:00.149-0500] {local_task_job_runner.py:245} INFO - ::endgroup::
